console/console-init/ui/src/components/UpdatePassword/UpdatePassword.tsx

```diff
@@ -25,16 +25,17 @@ export const UpdatePassword = () => {
   });
   const { state, dispatch } = useStoreContext();
   const { modalProps } = (state && state.modal) || {};
-  const { onClose, onConfirm } = modalProps || {};
+  const { onClose, onConfirm, additionallProps } = modalProps || {};
```

Comment:
Please provide some meaningful name here instead of additionalProps.

Reply:
> Please provide some meaningful name here instead of additionalProps.

We are passing multiple properties/porps which are not fixed. I think additionalProps name looks better. Suggest name if you have any. I'll take care in next PR.